packages/react-resizable-panels/src/hooks/useIsomorphicEffect.ts

```diff
@@ -3,6 +3,6 @@ import { useLayoutEffect_do_not_use_directly } from "../vendor/react";
 
 const useIsomorphicLayoutEffect = isBrowser
   ? useLayoutEffect_do_not_use_directly
-  : () => {};
+  : () => undefined;
```

Comment:
This change seems unexpected/unnecessary to me. Did ESLint actually complain about it? 🤔

Reply:
> An arrow function that returns undefined instead of an empty function body (that may seem like it returns an object)

FWIW `() => {}` does not return anything. It just defines an empty function, same as `function() {}`

If you wanted to return an object from an inline function, you'd need to wrap with parens `() => ({})`

Reply:
> FWIW `() => {}` does not return anything. It just defines an empty function, same as `function() {}`
> 
> If you wanted to return an object from an inline function, you'd need to wrap with parens `() => ({})`

Yes. I was trying to say it looks similar to returning an object.

Should we keep this version or revert?

Reply:
I don't feel strongly either way on this one. It just seemed unexpected.